pkg/signature/kms/kms.go

```diff
@@ -66,8 +65,9 @@ func Get(ctx context.Context, keyResourceID string, hashFunc crypto.Hash, opts .
 		}
 	}
 	sv, err := cliplugin.LoadSignerVerifier(ctx, keyResourceID, hashFunc, opts...)
-	if errors.Is(err, exec.ErrNotFound) {
```

Comment:
I just tried this with both sigstore and cosign tests. I think the better way is to change this line, and you can also edit the sigstore tests to cover this case.

```
	if errors.Is(err, exec.ErrNotFound) || errors.Is(err, cliplugin.ErrorInputKeyResourceID) {
		return nil, fmt.Errorf("%w: %w", &ProviderNotFoundError{ref: keyResourceID}, err)
	}
```